### PR TITLE
Add user authentication information

### PR DESCRIPTION
**Pull request by Amli**
Documentation update for the v2 API modification that adds the authentication attributes to the user object.

Wait for production release of https://github.com/opendatasoft/platform/pull/6391 to merge to deploy.

### CODE REVIEW

File: source/includes/_topics.md [hunk 1]
```diff
@@ -167,7 +167,10 @@ curl http://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXX/secu
       "last_name": "Last name",
       "email": "myuser@example.com",
       "is_active": true,
-      "is_ods": false
+      "is_ods": false,
+      "account_type": "global",
+      "display_name": "myuser",
+      "auth_provider_types": [],
```
Comment (Aquassaut) [inline source/includes/_topics.md]: The comma makes it invalid JSON

File: source/includes/_users.md [hunk 2]
```diff
@@ -63,6 +66,9 @@ Attribute | Description
 `first_name` <br> *string*      | The user's first name
 `last_name` <br> *string*       | The user's last name
 `email` <br> *string*           | The user's e-mail address
+`account_type` <br> *string*    | The user's account type.<br> Possible values are `global`, `linked` and `local`
+`display_name` <br> *string*    | The user's display name<br>simplified version of the `username`
+`auth_provider_types` <br> *array of string*    | The list of potential authentification providers type for this user.<br> Currently `saml` is the only available provider.
```
Comment (Aquassaut) [inline source/includes/_users.md]: I wouldn't say `potential`, even though I understand you mean that they could authenticate with either, but only if they choose so.

Comment (Amli) [reply source/includes/_users.md]: As the list is not really a complete list of the user options (we never mention `central` in the list), I guess I'll just drop the `potential` part.

File: source/includes/_topics.md (hunk 1 above)
Comment (Amli) [reply source/includes/_topics.md]: ✅

**Review by Aquassaut — APPROVED** [review]

**Comment by Amli** [comment]
thank you @Aquassaut 
@remilejeune your call for merging

**Pull request merged**